nmigen_boards/ecp5_5g_evn.py

```diff
@@ -15,20 +15,117 @@ class ECP55GEVNPlatform(LatticeECP5Platform):
     speed       = "8"
     default_clk = "clk12"
     default_rst = "rst"
+    def __init__(self, *, VCCIO1="LVCMOS25", VCCIO2="LVCMOS33", VCCIO6="LVCMOS33", VCCIO7="LVCMOS33", **kwargs):
```

Comment:
These names are a bit confusing--"LVCMOS25" is not a voltage on VCCIO1 but rather an IO standard. I think the constructor should take voltages as arguments--the Atlys board uses strings like `"2V5"`, so it'd be nice to be consistent here. Alternatively the constructor could take resistor positions instead, but it seems like voltage would be less confusing here.